vision/google/cloud/vision/_gax.py

```diff
@@ -30,24 +30,25 @@ def __init__(self, client=None):
         self._client = client
         self._annotator_client = image_annotator_client.ImageAnnotatorClient()
 
-    def annotate(self, image, features):
+    def annotate(self, images):
```

Comment:
It's not breaking in the sense of `_gax.py` being public, but it is breaking in the sense of arbitrarily changing the architecture